python/examples/join_streams.py

```diff
@@ -1,14 +1,15 @@
 """Merges messages from two senders based on timestamp.
 """
 
-import erdos
 import time
 from typing import Any
 
 from erdos.context import TwoInOneOutContext
 from erdos.operator import Source, TwoInOneOut
 from erdos.streams import WriteStream
 
+import erdos
```

Comment:
Could you verify that the `isort` action is functioning correctly? When I run `isort --check-only --diff --profile=black .` locally, this is moved above the `from erdos.* import ...` imports.

Reply:
Weird, running `isort --check-only --diff --profile black python/examples/join_streams.py` with `isort 5.10.1` after putting it above the `from erdos.* import ...` gives me this:

```python
ERROR: /Users/sukritkalra/PhD/erdos/python/examples/join_streams.py Imports are incorrectly sorted and/or formatted.
--- /Users/sukritkalra/PhD/erdos/python/examples/join_streams.py:before 2022-03-23 10:13:18.521570
+++ /Users/sukritkalra/PhD/erdos/python/examples/join_streams.py:after  2022-03-23 10:13:22.709842
@@ -4,12 +4,11 @@
 import time
 from typing import Any

-import erdos
-
 from erdos.context import TwoInOneOutContext
 from erdos.operator import Source, TwoInOneOut
 from erdos.streams import WriteStream

+import erdos


 class SendOp(Source):
```